src/Bicep.Core/Registry/Oci/OciArtifactResult.cs

```diff
@@ -30,6 +31,7 @@ public OciArtifactResult(BinaryData manifestBits, string manifestDigest, Immutab
 
         public string ManifestDigest { get; init; }
 
-        public IEnumerable<(string MediaType, BinaryData Data)> Layers { get; init; }
+        public IEnumerable<OciArtifactLayer> Layers { get; init; }
```

Comment:
Given that you're making this immutable in the constructor, may as well represent this as `ImmutableArray<OciArtifactLayer>`